src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,20 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+
+  <div className="movies">
+    {
+      movies.map(x => (
+        <MovieCard movie={x} />
+      ))
+    }
+  </div>
```

Comment:
```suggestion
  <div className="movies">
    {
      movies.map(movie => (
        <MovieCard movie={movie} key={movie.imdbId} />
      ))
    }
  </div>
```

also use semantic tags for the list, here ul instead of div